Include/internal/pycore_runtime.h

```diff
@@ -70,6 +71,7 @@ typedef struct _Py_DebugOffsets {
         uint64_t imports_modules;
         uint64_t sysdict;
         uint64_t builtins;
+        uint64_t _gil;
```

Comment:
This should probably be named `gil_runtime_state` and swapped down to be right above `gil_runtime_state_locked` and `gil_runtime_state_holder`, since it's the offset to the start of that structure:
```cpp
                ._gil = offsetof(PyInterpreterState, _gil), \
                .ceval_gil = offsetof(PyInterpreterState, ceval.gil), \
                .gil_runtime_state_locked = offsetof(PyInterpreterState, _gil.locked), \
                .gil_runtime_state_holder = offsetof(PyInterpreterState, _gil.last_holder), \
```